server/src/main/java/com/google/musicanalysis/api/naturallanguage/SentimentRequest.java

```diff
@@ -22,7 +22,9 @@ public SentimentRequest(String text) {
    * @return the JSON response as string
    */
   public String getResponse() throws MalformedURLException, IOException {
-    URL url = new URL("https://language.googleapis.com/v1/documents:analyzeSentiment?key=");
+    URL url =
+        new URL(
+            "https://language.googleapis.com/v1/documents:analyzeSentiment?key=");
```

Comment:
formatting looks a little weird - could you put 26-27 on the same line? (or does it exceed char limit)